### ISSUE TITLE
Compile error when using same TypeAdapter twice

### ISSUE THREAD

**Issue by jush** [rharter/auto-value-parcel#38]
I have the following class:
```Java
import com.google.android.gms.maps.model.LatLng;
...
@AutoValue
public abstract class RouteSegment implements Parcelable {
    @ParcelAdapter(LatLngTypeAdapter.class)
    public abstract LatLng startPoint();
    @ParcelAdapter(LatLngTypeAdapter.class)
    public abstract LatLng endPoint();
    ...
    public static class LatLngTypeAdapter implements
        com.ryanharter.auto.value.parcel.TypeAdapter<LatLng> {
        @Override
        public LatLng fromParcel(Parcel in) {
            return in.readParcelable(RouteSegment.class.getClassLoader());
        }

        @Override
        public void toParcel(LatLng value, Parcel dest) {
            dest.writeParcelable(value, 0);
        }
    }
}
```

That generates the following error:
`AutoValue_RouteSegment.java:14: error: variable latLngTypeAdapter is already defined in method createFromParcel(Parcel)`

If I remove the `ParcelAdapter` to `endPoint` I get the following error:
`error: AutoValue property endPoint is not a supported Parcelable type.`

How should `ParcelAdapter` be used in this scenario?

**Comment by solcott** [comment]
This was fixed by #34.  Will be out whenever the next release happens.

**Comment by rharter** [comment]
I've got one more issue to look at, but will probably have a release today.
